src/app/assumptions-and-limitation/assumptions-and-limitation.component.ts

```diff
@@ -40,6 +41,36 @@ export class AssumptionsAndLimitationComponent implements AfterContentInit {
     data: siteTypeInfo[purpose].averageMonthlyUserData,
   }));
   readonly locationCarbonInfo;
+  readonly deviceInfo = [
+    {
+      name: 'Laptop',
+      info: laptop,
+    },
+    {
+      name: 'Desktop',
+      info: desktop,
+    },
+    {
+      name: 'Server',
+      info: server,
+    },
+    {
+      name: 'Network',
+      info: network,
+    },
+    {
+      name: 'Mobile',
+      info: mobile,
+    },
+    {
+      name: 'Tablet',
+      info: tablet,
+    },
+    {
+      name: 'Monitor',
+      info: monitor,
+    },
+  ];
```

Comment:
I have thought about a Map, since it preserves order, which was my initial reason for using an array instead. My thinking was that down the line we might have a service that can provide a list of all devices (which internally stores a Map), and we could map it to add the user facing description.

Reply:
Ok, let's stick with the array.